PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -164,12 +162,11 @@
         softId = Var("passed('SoftCutBasedId')",bool,doc="soft cut-based ID"),
         softMvaId = Var("passed('SoftMvaId')",bool,doc="soft MVA ID"),
         softMva = Var("softMvaValue()",float,doc="soft MVA ID score",precision=6),
-        softMvaRun3 = Var("softMvaRun3Value()",float,doc="soft MVA Run3 ID score",precision=6),
```

Comment:
Is the removal of `softMvaRun3` intended?

Reply:
Indeed it was not. This is fixed in the new commit